roles/check_changelog_fragment/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+# RULES
+# * a pull-request needs a new changelog fragment
+# * unless:
+#   * it is a "New plugin" pull-request
+#   * unless:
+#     * the PR modifies existing file(s)
+#     * the PR deletes existing file(s)
+#     * the PR renames existing file(s)
+
+
+# Rebasing ensures diffs are related to the PR, and only to the PR. If the task
```

Comment:
I finally had some time to look at this, there isn't a need to rebase here. We have dedicated zuul-mergers that run merge operations, if the merge fails the job won't run. So, lets delete this.

Reply:
ah, ok, that's fine, I'll remove it